MobileCenterDistribute/MobileCenterDistribute/Internals/MSDistributeAppDelegate.m

```diff
@@ -0,0 +1,39 @@
+#import "MSAppDelegateForwarder.h"
+#import "MSDistributeAppDelegate.h"
+#import "MSDistribute.h"
+
+@implementation MSDistributeAppDelegate
+
+#pragma mark - MSAppDelegate
+
+-(BOOL)application:(__attribute__((unused)) UIApplication *)app openURL:(NSURL *)url sourceApplication:(__attribute__((unused)) NSString *)sourceApplication annotation:(__attribute__((unused)) id)annotation returnedValue:(BOOL)returnedValue{
+  return [self openURL:url returnedValue:returnedValue];
+}
+
+-(BOOL)application:(__attribute__((unused)) UIApplication *)app openURL:(NSURL *)url options:(__attribute__((unused)) NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options returnedValue:(BOOL)returnedValue{
+  return [self openURL:url returnedValue:returnedValue];
+}
+
+#pragma mark - Private
+
+-(BOOL)openURL:(NSURL *)url returnedValue:(BOOL)returnedValue{
```

Comment:
Can you run reformat one more time? Make sure you run formatter against all source files in the PR.